nix/default.nix

```diff
@@ -228,7 +228,7 @@ in rec {
       sha256 = "ocz3CJFf+diThmocrgSnhWW/fjuRLLyCxwUKl3Cm7WA=";
     };
   });
-  elasticsearch-home = "/var/lib/elasticsearch";
+  elasticsearch-home = "~/monocle-elasticsearch";
```

Comment:
Perhaps we should use xdg path, e.g. `~/.local/share/monocle/elasticsearch-home`